packages/renderer/src/lib/markdown/micromark-button-directive.ts

```diff
@@ -34,19 +34,32 @@ export function button(d: any) {
     return false;
   }
 
-  this.tag(
-    '<a class="px-4 py-[6px] rounded-[4px] text-white text-[13px] whitespace-nowrap bg-purple-600 hover:bg-purple-500 no-underline"',
-  );
+  // create button html if it has command attribute
+  if (d.attributes && 'command' in d.attributes) {
+    this.tag(
+      '<button class="px-4 py-[6px] rounded-[4px] text-white text-[13px] whitespace-nowrap bg-purple-600 hover:bg-purple-500 no-underline" onclick="window.executeOnboardingButtonCommand(\'' +
+        this.encode(d.attributes.command) +
+        '\')">',
```

Comment:
it's not only an onboarding button so I would not call it 'executeOnboardingButtonCommand'

it can be in settings, etc, anywhere we have a custom markdown